index.html

```diff
@@ -159,6 +159,12 @@ <h1>YouTubers' Birthdays</h1>
 					<td>April 20, 1992</td>
 					<td>213 million</td>
 				</tr>
+				<tr>
+					<td> Tech</td>
+					<td><a href="https://www.youtube.com/@TiffInTech/featured">Tiff in Tech</a></td>
+					<td>Birthday Unavailable</td>
```

Comment:
The point of this project is to list YouTubers birthday so this field is a must. I tried searching for Tiff's birthday, but can't find it either.

I suggest asking her community via discord or finding her birthday via socials. Till her birthday is found, I cannot merge this, sorry.

Reply:
Ok thanks